Remotion/ObjectBinding/Web.UnitTests/UI/Controls/ReflectionBusinessObjectWebUIService.cs

```diff
@@ -57,7 +58,7 @@ public string GetToolTip (IBusinessObject obj)
       ArgumentUtility.CheckNotNull("businessObjectClass", businessObjectClass);
 
       return new HelpInfo(
-          "#",
+          SafeServiceLocator.Current.GetInstance<IFallbackNavigationUrlProvider>().GetURL(),
```

Comment:
Hmm... here we can (should) just use "fakeFallbackUrl" or not change it at all. It's a hardcoded value we inject and unless this is used as a reference value for comparison with actual results generated during the tests, the URL value is of no import.